pkg/flag/report_flags.go

```diff
@@ -155,6 +155,10 @@ func (f *ReportFlagGroup) ToOptions(out io.Writer) (ReportOptions, error) {
 	listAllPkgs := getBool(f.ListAllPkgs)
 	output := getString(f.Output)
 
+	if err := report.ValidateFormat(format); err != nil {
+		return ReportOptions{}, err
```

Comment:
```suggestion
		return ReportOptions{}, xerrors.Errorf("unknown format: %v", format)
```

Reply:
@knqyf263 Thanks for your suggestions. I also thought replacing independent strings with something like `SupportedSBOMFormats` is a better solution.

Reply:
>replace the original strings with the slice

What does it look like? I thought we would add the slice, but I would like to hear another approach.

Reply:
Still learning golang, I wrongly thought that I can treat Golang's slice as Cshap's enum which can be specified by switch-case or test if a value belongs to one of the enum. Anyway, I finally implemented your suggestion.